lib/box/client.go

```diff
@@ -135,8 +136,19 @@ func (err ExitError) Error() string {
 func dial(socketPath string) (net.Conn, error) {
 	conn, err := net.Dial("unix", socketPath)
 	if err != nil {
```

Comment:
I can't see an easy Go-like way to check whether the error returned by net.Dial contains "permission denied" or not - do you have any advice?